src/components/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import classes from 'classnames';
+import md5 from 'md5';
+import { trim } from 'lodash';
+/**
+ * Displays user profile avatar image with/withour user name
+ */
+const Avatar = ({ className, size, userName, imgUrl, userEmail }) => {
+  const gravatarUrl = userEmail
+    ? `https://www.gravatar.com/avatar/${md5(trim(userEmail.toLowerCase()))}?s=${size}`
+    : imgUrl;
+  const avatarClass = classes('Avatar', className);
+  const avatarSize =
+    size >= 20 && size <= 200 ? { height: size, width: size } : { height: 40, width: 40 };
+  const avatarLetterSize =
```

Comment:
change to `avatarSize` as this actually dictates the size of the entire component and not just the type

Reply:
I changed it to 
```
const avatarLetterSize = size >= 20 && size <= 200 ? { fontSize: size * 0.8 } : { fontSize: 32 };
```
and this would only to apply to the letter initial inside avatar.